### PR TITLE
Fix error "Array to string conversion" on multi language website

### PR DESCRIPTION
**Pull request by RubenMartins**
Fix error "Array to string conversion" on multi language website.

### CODE REVIEW

File: statsproduct.php [hunk 1]
```diff
@@ -157,6 +157,7 @@ public function hookAdminStatsModules()
     {
         $id_category = (int)Tools::getValue('id_category');
         $currency = Context::getContext()->currency;
+	$id_lang = $this->context->language->id;
```
Comment (PierreRambaud) [inline statsproduct.php]: Don't mix tabs and spaces please :)

Comment (RubenMartins) [reply statsproduct.php]: Fixed it in https://github.com/PrestaShop/statsproduct/pull/8/commits/36e97bc9ce25ad3c20621340d37c0b53c9deea1a

**Comment by RubenMartins** [comment]
Fixed it in https://github.com/PrestaShop/statsproduct/pull/8/commits/36e97bc9ce25ad3c20621340d37c0b53c9deea1a @PierreRambaud

**Comment by Progi1984** [comment]
@RubenMartins  Thank you for your contribution. :hugs: 

Could you give us a scenario which permits to test the bug before and after this PR ?

Thanks :+1:

**Comment by RubenMartins** [comment]
> @RubenMartins Thank you for your contribution. 🤗
> 
> Could you give us a scenario which permits to test the bug before and after this PR ?
> 
> Thanks 👍

I will need to re-verify, as this as a long time ago. But from what I remember this happened when as set to multi-language and trying to check the Products Stats that trough this error. I'm not sure if it was only in error_log and visible showing nothing or if it was actually showing this error to the user on the backend.

**Comment by Progi1984** [comment]
> > @RubenMartins Thank you for your contribution. hugs
> > Could you give us a scenario which permits to test the bug before and after this PR ?
> > Thanks +1
> 
> I will need to re-verify, as this as a long time ago. But from what I remember this happened when as set to multi-language and trying to check the Products Stats that trough this error. I'm not sure if it was only in error_log and visible showing nothing or if it was actually showing this error to the user on the backend.
> 
> And this commit fixed the issue with showing the sats in a multi-language store or even singlelanguage.

Thanks for the scenario. Could you check and confirm it us ? Thanks :)

**Comment by sarahdib** [comment]
Hello @RubenMartins 
Did you find the scenario ? 
I try to reproduce with the information you gave us but I didn't reproduce.

Thank you

**Comment by sarahdib** [comment]
@RubenMartins can we close this PR ? We can't reproduce the first bug.

**Comment by sarahdib** [comment]
@RubenMartins 
I close this pr since I can't reproduce the issue. 
Feel free to re-open it if you remember the scenario.

Have a nice day

**Pull request closed without merging**